.github/workflows/meson.yml

```diff
@@ -0,0 +1,44 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  # Customize the Meson build type here (plain,debug,debugoptimized,release,minsize,custom)
+  BUILD_TYPE: release
+
+jobs:
+  build:
+    name: Ubuntu 20.04
+    # You can convert this to a matrix build if you need cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
+      #with:
+      #  cache: 'pip'
+    
+    - name: Install Buildsystem
+      run: |
+        pip install meson==0.53.0
+        sudo apt-get install -y ninja-build
+    
+    - name: Install Libtasn1
+      run: sudo apt-get install -y libtasn1-dev libtasn1-bin
+
+    - name: Configure Meson
+      run: meson setup ${{github.workspace}}/build --buildtype ${{env.BUILD_TYPE}}
+
+    - name: Build
+      run: ninja -C ${{github.workspace}}/build
```

Comment:
Could you add a ninja dist too ?

If so, it need to be done after a first ninja, that will generate the spnego_asn1.c file, so that it can be included in the tar.